--- ipxe.spec
+++ ipxe.spec
@@ -40,6 +40,7 @@
 URL:     http://ipxe.org/
 
 Source:  %forgesource
+Source1: thinstation-autoexec.ipxe
 
 # Enable IPv6 for qemu's config
 # Sent upstream: http://lists.ipxe.org/pipermail/ipxe-devel/2015-November/004494.html
@@ -129,7 +130,7 @@
 make_ipxe bin-x86_64-efi/snponly.efi
 
 make_ipxe ISOLINUX_BIN=/usr/share/syslinux/isolinux.bin \
-    bin/undionly.kpxe
+    bin/undionly.kpxe EMBED=%{SOURCE1}
 
 # build roms with efi support for qemu
 mkdir bin-combined
